src/main/java/com/codingParkFun/rmsbackend/controller/BatchController.java

```diff
@@ -26,29 +26,26 @@
 
 	@PostMapping
 	@ResponseStatus(HttpStatus.CREATED)
-	public void createGroup(@PathVariable long schoolId, @RequestBody Batch group) {
-		
+	public void createGroup(@PathVariable String schoolId, @RequestBody Batch group) {
+		long id = Long.parseLong( schoolId.substring( 1,schoolId.length()-1 ) );
```

Comment:
Why substring is required ? What characther are you getting as first charcter ?

Reply:
String will be like this {id}.

Reply:
Ohh I think something wrong will check.